package.json

```diff
@@ -42,11 +50,19 @@
     "underscore": "^1.9.1"
   },
   "devDependencies": {
+    "@babel/cli": "^7.6.4",
+    "@babel/core": "^7.6.4",
+    "@babel/node": "^7.6.3",
+    "@babel/preset-env": "^7.6.3",
+    "@babel/register": "^7.6.2",
```

Comment:
all of this are packages on `dependencies`, why include them on `devDependencies`

Reply:
Let me fix that.

Reply:
@b0nbon1 I think they are all for development purposes only, so should be in `devDependencies`

Reply:
I'll move them to devDependencies.